src/main/java/Main.java

```diff
@@ -19,8 +17,8 @@
 
 public class Main {
     private static final Scanner scanner = new Scanner(System.in);
-    private static final OffenseFacade offenseFacade = new OffenseFacadeImpl(new OffenseDaoImpl());
-    private static final CommunityServiceFacade communityServiceFacade = new CommunityServiceFacadeImpl(new CommunityServiceDaoImpl());
+    private static final OffenseFacade offenseFacade = new OffenseFacadeImpl();
```

Comment:
why are these objects final?

Reply:
About po this one. I double check po yung last merge naka final na po sya. Tinanggal ko lng po yung parameter for unit testing.